project/src/components/choosen-tab/choosen-tab.tsx

```diff
@@ -0,0 +1,27 @@
+import NotFound from '../404-screen/404-screen';
+import FilmOverview from '../fiim-overview/film-overview';
+import FilmDetails from '../film-details/film-details';
+import FilmReviews from '../film-reviews/film-reviews';
+
+
+export  const choosenTabRender=(currentTab: number): JSX.Element=> {
+
+
+  switch (currentTab) {
+    case 1:
+      return (<FilmOverview />);
+      break;
+    case 2:
+      return (<FilmDetails />);
+      break;
+    case 3:
+      return (<FilmReviews />);
+      break;
+    default: return <NotFound />;
+
+
+  }
```

Comment:
`switch`-ей, по возможности следует избегать. _свечки пожароопасны_ :)
```suggestion
//где-то наверху
const settings = new Map([
[1,<FilmOverview />],
[2,<FilmDetails />],
[3,<FilmReviews />],
]);
// и где-то в этом месте 

return settings.get(currentTab)||<NotFound />
```